libs/utils/analysis/frequency_analysis.py

```diff
@@ -110,6 +110,70 @@ def _dfg_cluster_frequency_residency(self, cluster, total=True):
             return residency.total
         return residency.active
 
+    def _dfg_frequency_transitions(self, entity):
+        """
+        Compute number of frequency transitions of a given entity. Entity can
+        be a CPU or a cluster name. Notice that we assume that frequency is
+        scaled at cluster level, therefore we always consider the first CPU of
+        a cluster for this computation.
+
+        Requires cpu_frequency events to be available in the trace.
+
+        :param entity: a CPU ID or a cluster name
+        :type entity: int or str
+
+        :returns: :mod:`pandas.DataFrame` - number of frequency transitions
+        """
+        if not self._trace.hasEvents('cpu_frequency'):
+            self._log.warn('Events [cpu_frequency] not found, '
+                           'cannot compute frequency transitions!')
+            return None
+
+        if isinstance(entity, str):
+            cpu = self._platform['clusters'][entity.lower()][0]
+        else:
+            cpu = entity
+
+        freq_df = self._dfg_trace_event('cpu_frequency')
+        cpu_freqs = freq_df[freq_df.cpu == cpu].frequency
```

Comment:
We are trying to maintain a "_df" suffix for all DataFrames.
This is still a DataFrame, thus we should keep the "_df" suffix... or just reuse `freq_df`

Reply:
`cpu_freqs` is actually a series that is why I didn't put the `_df` suffix. Should I use `_s`?